[2023-03-07 14:37:20,307] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: meu_primeiro_dag.Executa_api scheduled__2023-03-06T01:01:00+00:00 [queued]>
[2023-03-07 14:37:20,312] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: meu_primeiro_dag.Executa_api scheduled__2023-03-06T01:01:00+00:00 [queued]>
[2023-03-07 14:37:20,312] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-03-07 14:37:20,312] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-03-07 14:37:20,312] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-03-07 14:37:20,323] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): Executa_api> on 2023-03-06 01:01:00+00:00
[2023-03-07 14:37:20,325] {standard_task_runner.py:52} INFO - Started process 64310 to run task
[2023-03-07 14:37:20,327] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'meu_primeiro_dag', 'Executa_api', 'scheduled__2023-03-06T01:01:00+00:00', '--job-id', '84', '--raw', '--subdir', 'DAGS_FOLDER/cotacao_dolar.py', '--cfg-path', '/tmp/tmp20_51817', '--error-file', '/tmp/tmp6yda6n4v']
[2023-03-07 14:37:20,327] {standard_task_runner.py:80} INFO - Job 84: Subtask Executa_api
[2023-03-07 14:37:20,355] {task_command.py:370} INFO - Running <TaskInstance: meu_primeiro_dag.Executa_api scheduled__2023-03-06T01:01:00+00:00 [running]> on host veripc-System-Product-Name
[2023-03-07 14:37:20,472] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=meu_primeiro_dag
AIRFLOW_CTX_TASK_ID=Executa_api
AIRFLOW_CTX_EXECUTION_DATE=2023-03-06T01:01:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-03-06T01:01:00+00:00
[2023-03-07 14:37:20,473] {python.py:173} INFO - Done. Returned value was: None
[2023-03-07 14:37:20,476] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=meu_primeiro_dag, task_id=Executa_api, execution_date=20230306T010100, start_date=20230307T173720, end_date=20230307T173720
[2023-03-07 14:37:20,499] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-03-07 14:37:20,523] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
